src/index.d.ts

```diff
@@ -6,16 +6,22 @@ declare module "jest-fetch-mock" {
     headers?: Object;
   }
 
-  const fn: {
+  interface MockContext {
+    calls: any[][];
+    instances: any[];
+  }
+
+  interface Fetch {
     (input?: string | Request, init?: RequestInit): Promise<Response>;
-    mockResponse(body: string, init?: MockParams): void;
-    mockResponseOnce(body: string, init?: MockParams): void;
-    mockResponses(responses: Array<{body: string, init?: MockParams}>): void;
-    resetMocks(): void;
-    mockReject(error?: Error): void;
-    mockRejectOnce(error?: Error): void;
+    mockResponse(body: string, init?: MockParams): Fetch;
+    mockResponseOnce(body: string, init?: MockParams): Fetch;
+    mockResponses(responses: Array<{body: string, init?: MockParams}>): Fetch;
+    mockReject(error?: Error): Fetch;
+    mockRejectOnce(error?: Error): Fetch;
     resetMocks(): void;
+    mock: MockContext;
   }
 
+  const fn: Fetch;
```

Comment:
If the type of `fn` was `Fetch & jest.MockInstance<any>` then would you not need `MockContext` and `mock` defined above?

Reply:
That is true. Creating a new PR